### DISCUSSION TITLE
GitHub Co-Pilot Not Logging Into Rider (Jetbrains)

### DISCUSSION THREAD

**Discussion by zakkosma** [community/community#20453] [Copilot Conversations]
Hi,

My name is Zak. I tried setting up GitHub Co-Pilot on a fresh install of Jetbrains Rider IDE 2022.1

I go through the typical login process and I am able to authorize Copilot on the website.

When I go back to Jetbrains it tells me to login into Github copilot.

![image](https://user-images.githubusercontent.com/8184272/178807552-d390aabe-fe85-4bf0-ad69-0a0bfcea9a3d.png)

Any ideas on how I can get this resolved?

**Comment by davecheney** [comment]
@zakkosma i see no requests from your account to the authentication service. Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?

**Comment by smstromb** [comment]
@zakkosma I had the same issue as you. When I went to github -> settings -> codepilot I saw a notification saying that my github codepilot setup was incomplete. After selecting an option for the prompt `GitHub Copilot can allow or block suggestions matching public code.`, the IDE was able to progress past login.
